recipes/sca_e_projections_by_boro/build.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TEMP TABLE tmp as (
+    SELECT CASE 
```

Comment:
Why not just combine build.sql and build.py
we can just do a `pd.read_sql(<build.sql>)`